troposphere/static/js/components/projects/resources/instance/details/sections/details/IpAddress.jsx

```diff
@@ -21,7 +21,7 @@ export default React.createClass({
         return (
         <ResourceDetail label="IP Address">
             {address}
-            <CopyButton text={ address }/>
+            {address !== "N/A" ? <CopyButton text={ address }/> : null}
```

Comment:
if someone changes the missing address text to "<no ip>", then copy button behavior won't be affected.